app/src/main/res/drawable-mcc240/background_maxheight_sign.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Sweden SE -->
+<!-- TODO: this and the other two country-dependent files don't work -->
```

Comment:
> Actually refactor `getCurrentCountryResources` to `getResources` (override)

But this can't be done because `getResources` is `final` and thus can't be overridden... Or am I missing something?
https://developer.android.com/reference/android/content/Context#getResources()

Reply:
Which means
```
LayoutInflater.from(new ContextWrapper(getContext())
{
	@Override public Resources getResources()
	{
		return AbstractQuestAnswerFragment.this.getCurrentCountryResources();
	}
});
```